internal/rpc/jsonrpc/rpcserverhelp.go

```diff
@@ -75,6 +75,7 @@ func helpDescsEnUS() map[string]string {
 		"sweepaccount":            "sweepaccount \"sourceaccount\" \"destinationaddress\" (requiredconfirmations feeperkb)\n\nMoves as much value as possible in a transaction from an account.\n\n\nArguments:\n1. sourceaccount         (string, required)  The account to be swept.\n2. destinationaddress    (string, required)  The destination address to pay to.\n3. requiredconfirmations (numeric, optional) The minimum utxo confirmation requirement (optional).\n4. feeperkb              (numeric, optional) The minimum relay fee policy (optional).\n\nResult:\n{\n \"unsignedtransaction\": \"value\",     (string)  The hex encoded string of the unsigned transaction.\n \"totalpreviousoutputamount\": n.nnn, (numeric) The total transaction input amount.\n \"totaloutputamount\": n.nnn,         (numeric) The total transaction output amount.\n \"estimatedsignedsize\": n,           (numeric) The estimated size of the transaction when signed.\n}                                    \n",
 		"ticketsforaddress":       "ticketsforaddress \"address\"\n\nRequest all the tickets for an address.\n\nArguments:\n1. address (string, required) Address to look for.\n\nResult:\ntrue|false (boolean) Tickets owned by the specified address.\n",
+		"ticketinfo":              "ticketinfo\n\nReturns details of each wallet ticket transaction\n\nArguments:\nNone\n\nResult:\n{\n \"hash\": \"value\",               (string)          Transaction hash of the ticket\n \"cost\": n.nnn,                 (numeric)         Amount paid to purchase the ticket; this may be greater than the ticket price at time of purchase\n \"votingaddress\": \"value\",      (string)          Address of 0th output, which describes the requirements to spend the ticket\n \"status\": \"value\",             (string)          Description of ticket status (unknown, unmined, immature, mature, live, voted, missed, expired, unspent, revoked)\n \"blockhash\": \"value\",          (string)          Hash of block ticket is mined in\n \"blockheight\": n,              (numeric)         Height of block ticket is mined in\n \"vote\": \"value\",               (string)          Transaction hash of vote which spends the ticket\n \"revocation\": \"value\",         (string)          Transaction hash of revocation which spends the ticket\n \"generated\": n.nnn,            (numeric)         How much coins were generated by a vote\n \"choices\": [{                  (array of object) Vote preferences set for the ticket\n  \"agendaid\": \"value\",          (string)          The ID for the agenda the choice concerns\n  \"agendadescription\": \"value\", (string)          A description of the agenda the choice concerns\n  \"choiceid\": \"value\",          (string)          The ID of the current choice for this agenda\n  \"choicedescription\": \"value\", (string)          A description of the current choice for this agenda\n },...],                                          \n}                               \n",
```

Comment:
`agendadescription` and `choicedescripion` are mentioned here but they are actually absent from the response.

Reply:
Also this seems to indicate that the result is just a single ticket object, rather than an array.

Reply:
It's not obvious why ticket status may be returned as `"unknown"` - I had to dig through the code to find that it can be unknown if SPV mode is enabled (and thus dcrd cannot be queried).

Might be worth a quick mention in the doco to save others having to dive into the code in future.

Reply:
we'd need a new type for the choice ids without any description fields for that to not show in the help.